app/policies/match_policy.rb

```diff
@@ -0,0 +1,5 @@
+class MatchPolicy < ApplicationPolicy
+  def update?
+    admin? || @record.contacted_users.include?(@user)
```

Comment:
`@record.contacted_users.include?(@user)`, c’est la même chose que `@record.expert.in?(@user.experts)`, en plus clair. :+1: